libs/application/templates/official-journal-of-iceland/src/components/htmlEditor/HTMLEditor.tsx

```
@@ -42,6 +44,12 @@ export const HTMLEditor = ({
     }
   }, [value, readOnly])
 
+  const handleChange = (value: HTMLText) => {
+    debounce(() => {
+      onChange && onChange(value)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)